rabbitmq/meta/config.yml

```diff
@@ -0,0 +1,7 @@
+config:
+  {%- if pillar.rabbitmq.server is defined %}
+  {%- from "rabbitmq/map.jinja" import server with context %}
```

Comment:
Using the same approach like: https://github.com/tcpcloud/salt-formula-galera/commit/4bca6a38e1838f504946ba53eb9e31010b4b39b8. Trying to keep it same.